tests/yam/validate/validate_migration_logs.pm

```diff
@@ -21,10 +21,16 @@ sub run {
     script_run("tar czvf /tmp/udev_rules.tar.gz /etc/udev/rules.d/*", {timeout => 60});
     upload_logs("/tmp/udev_rules.tar.gz", failok => 1);
 
+    # fatal migration errors that lead to failed test run
     if (script_run("cat /var/log/distro_migration.log | grep -i -E \"migration failed|aborting migration\"") == 0) {
         record_info("Migration failed", script_output("cat /var/log/distro_migration.log | grep -i -E \"migration failed|aborting migration\" -B50"), result => 'fail');
         die("Migration failed");
     }
+
+    # other non-fatal errors in migration log
+    if (script_run("cat /var/log/distro_migration.log | grep -iE \"failed|failure|error\" | grep -ivE \"\\-errors?|errors?\\-|error=''\"") == 0) {
+        record_info("Error in migration", script_output("cat /var/log/distro_migration.log | grep -ivE \"\\-errors?|errors?\\-|error=''\" | grep -iE \"failed|failure|error\" -B50"), result => 'fail');
```

Comment:
It only displays "Error in migratio" due to this field only admit certain number of character. "Migration" doesn't provide any extra information as we are running this test module for migration logs, what about "Minor errors" ?